cdap-distributions/src/hdinsight/createUiDef.json

```diff
@@ -0,0 +1,13 @@
+{
+    "handler": "Microsoft.HDInsight",
+    "version": "0.0.1-preview",
+    "clusterFilters": {
+        "types": [
+            "HBase"
+        ],
+        "tiers": [
+            "Standard"
+        ],
+        "versions": [ "3.4" ]
```

Comment:
Given that this is on one line, what about making the two previous entries? (types, tiers). 

Though I like the four-character spacing, we seem to be using two for our other JSON. I'd argue to leave it as-is, but if you want to match...